js/app.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+(function(){
+  angular
+    .module("radio", [
+      "ui.router",
+      "songs"
+    ])
+    .config([
+    "$stateProvider",
+    RouterFunction
+    ]);
+
+  function RouterFunction($stateProvider){
+    $stateProvider
+    .state("songIndex", {
+      url: "/songs",
+      templateUrl: "js/songs/index.html",
+      controller: "songIndexController",
+      controllerAs: "songIndexViewModel"
+    })
+    .state("songNew", {
+      url: "/songs/new",
```

Comment:
Great job with your RouterFunction here! I also liked that you added a `"root"` state.